### PR TITLE
Fix stylesheet HTML snippet for external renderers documentation

### PR DESCRIPTION
**Pull request by sbannier**
The documentation is missing the rel attribute. Neither Firefox nor Chrome did use the linked file as CSS if rel="stylesheet" is not set.

The problem is described in issue #22434.

### CODE REVIEW

File: docs/content/doc/advanced/external-renderers.en-us.md [hunk 1]
```diff
@@ -192,5 +192,5 @@ And so you could write some CSS:
 Add your stylesheet to your custom directory e.g `custom/public/css/my-style-XXXXX.css` and import it using a custom header file `custom/templates/custom/header.tmpl`:
 
 ```html
-<link type="text/css" href="{{AppSubUrl}}/assets/css/my-style-XXXXX.css" />
+<link rel="stylesheet" type="text/css" href="{{AppSubUrl}}/assets/css/my-style-XXXXX.css" />
```
Comment (silverwind) [inline docs/content/doc/advanced/external-renderers.en-us.md]:
```suggestion
<link rel="stylesheet" href="{{AppSubUrl}}/assets/css/my-style-XXXXX.css" />
```

It's the default type.

**Review by silverwind — APPROVED** [review]
lgtm after suggestion.

**Comment by sbannier** [comment]
Thanks for the feedback, I changed the code as proposed by you. Shall I squash both commits or is the PR good the way it is now?

**Comment by silverwind** [comment]
Not necessary as we squash on merge.

**Review by zeripath — APPROVED** [review]

**Pull request merged**